### PR TITLE
cloud: add release notes for 0214

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)
add cloud release notes

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,22 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**General changes**
+
+- Support decreasing the number of TiKV and TiFlash nodes to scale in a TiDB Dedicated Tier cluster.
+
+    You can decrease the node number [via the TiDB Cloud console](/tidb-cloud/scale-tidb-cluster.md#decrease-node-number) or [via the TiDB Cloud API (beta)](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster).
+
+**Console changes**
+
+- Introduce the **Monitoring** page for Serverless Tier clusters.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:21]:
```suggestion
- Introduce the **Monitoring** page for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,22 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## February 14, 2023
+
+**General changes**
+
+- Support decreasing the number of TiKV and TiFlash nodes to scale in a TiDB Dedicated Tier cluster.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Support decreasing the number of TiKV and TiFlash nodes to scale in a TiDB [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) cluster.
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**